src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -55,14 +72,20 @@ function setup {
         rm -rf $CW_DIR/codewind-workspace/*
```

Comment:
Can we do the similar thing for Che (eg. remove /projects), because we have `run` mode which we don't reinstall Codewind on Che and we may have projects exist in `/projects`

Reply:
I dont think we can do that in Che 🤔 

If we remove `/projects`, then we have to re-create workspace..  If we re-create workspace then all user projects will be lost.

Reply:
In that case, I think we have to skip this for Kube